apps/src/MusicController.js

```diff
@@ -89,9 +94,16 @@ module.exports = MusicController;
 function buildTrackData(trackDefinitions, assetUrl) {
   trackDefinitions = utils.valueOr(trackDefinitions, []);
   return trackDefinitions.map(function (trackDef) {
+
+    var assetUrls = [];
+    assetUrls.push(assetUrl(trackDef.name + '.mp3'));
+    if (trackDef.hasOgg) {
+      assetUrls.push(assetUrl(trackDef.name + '.ogg'));
+    }
```

Comment:
Seems good.  We probably should have gone back and put in ogg tracks for Star Wars too, but I'm not sure which browsers can play OGG and not MP3.

Please add the `hasOgg` to the MusicTrackDefinition typedef on line 27 of this file.